flink-connectors/flink-connector-elasticsearch-base/src/main/java/org/apache/flink/connector/elasticsearch/sink/ElasticsearchSinkBuilderBase.java

```diff
@@ -80,15 +76,16 @@
      * Sets the emitter which is invoked on every record to convert it to Elasticsearch actions.
      *
      * @param emitter to process records into Elasticsearch actions.
-     * @return {@link ElasticsearchSinkBuilder}
```

Comment:
replace everything with `@return this builder`. You usually link only to external types. In this case, the return type is actually a specific builder and not just base; so it's inaccurate.